nltk/tokenize/texttiling.py

```diff
@@ -171,19 +174,19 @@ def blk_frq(tok, block):
                 window_size = self.k
 
             b1 = [ts.index
-                  for ts in tokseqs[curr_gap-window_size+1 : curr_gap+1]]
+                  for ts in tokseqs[curr_gap-window_size+1:curr_gap+1]]
```

Comment:
It's un-PEP8 but maybe keeping the spaces around the colon will help in code reading. But I think it's fine to remove it too and say with the edited version.